build_scripts/dependencies_static_analysis_test_cov.py

```diff
@@ -0,0 +1,126 @@
+"""Koninklijke Philips N.V., 2019 - 2020. All rights reserved.
+build script to building the similarity tool"""
+import platform
+from subprocess_calls import call_subprocess
+from install_dependencies import install_pip
+
+
+def install_aspell():
+    """
+    Installs Aspell and configure for english
+    """
+    if str(platform.system()).upper() == "LINUX":
+        call_subprocess("sudo apt-get update -qq")
+        call_subprocess("sudo apt-get install --assume-yes aspell aspell-en")
+        print("Stage Install aspell -- COMPLETED & PASSED --")
+    else:
+        print("Please install and configure Aspell for english")
+
+
+def install_npm_packages():
+    """
+    Installs jscpd and configure for english
+    """
+    if str(platform.system()).upper() == "LINUX":
+        call_subprocess("sudo npm install")
+        call_subprocess("sudo npm install -g jscpd@3.2.1")
+        call_subprocess("sudo npm i -g yaml-lint@1.2.4")
+        call_subprocess("sudo npm i -g markdownlint-cli@0.23.1")
+        print("Stage Install jscpd, markdownlint & ymllint -- COMPLETED & PASSED --")
+    else:
+        print("Please install and configure jscpd, markdownlint & ymllint")
+
+
+def check_lint():
+    """
+    function check the repo for any python linting errors
+    """
+    call_subprocess("python3 -m pylint similarity_processor/ test/ build_scripts/ ")
+    print("Stage linting -- COMPLETED & PASSED  --")
+
+
+def check_yml_linting():
+    """
+    function check the repo for any yml linting errors
+    """
+    call_subprocess("yamllint .github/workflows/*.yml ")
+    print("Stage linting yml -- COMPLETED & PASSED  --")
+
+
+def check_md_linting():
+    """
+    function check the repo for any yml linting errors
+    """
+    print("----TODO----- Markdown linting")
+    # call_subprocess("markdownlint *.md ")
+    # print("Stage linting md files -- COMPLETED & PASSED  --")
```

Comment:
Work in progress?

Reply:
yes, i would like to take this as part of separate checkin as it will call for a refactor of readme.md. I was able to get output locally on linting the md files. Just that i would like to refactor readme with needed more-info